### ISSUE TITLE
Change facility - Missing Device plugin and Learn-only text after merging a learner

### ISSUE THREAD

**Issue by pcenov** [learningequality/kolibri#10680]
## Observed behavior
After merging a learner the Device plugin is missing and there is no indication that this is a Learn-only device.
I'm creating this as a follow-up issue to comments I've made in what are now merged PRs. 

## Expected behavior
The Device plugin should be displayed with an access to the Channels page.
The text 'Learn-only device Coach and admin features not available' should be displayed.

## Steps to reproduce the issue
1. Install the latest develop build
2. Setup a device through the 'On my own' flow and create a learner user
3. Sign in as the learner and migrate to a different device in the network

## Video

https://github.com/learningequality/kolibri/assets/79847249/6ca93ae3-3e8a-4a66-96ff-6448e0c3190c

## Logs
[LearnerLogs.zip](https://github.com/learningequality/kolibri/files/11451933/LearnerLogs.zip)

## Usage Details
Ubuntu 22  - Chrome, Firefox

**Comment by pcenov** [comment]
@radinamatic

**Comment by rtibbles** [comment]
This is incorrect.

If the learner account from this device is migrated to a different facility, then it will not become a super admin in the process. So it is not expected that they would have access to the Device plugin, nor that any warning would be needed to say that they cannot access things they are not expected to access.

If our gherkin stories say otherwise, we need to update them.

**Comment by pcenov** [comment]
@rtibbles this makes sense, I suppose in that case since they don't have access to Device > Channels they will just rely on automatic syncing to get content delivered to their device, right?

**Comment by rtibbles** [comment]
Yes - and there is still the original super user on the device that was made during setup, who can do these things as well.

**Comment by pcenov** [comment]
Closing as this is the intended behavior.
